src/main/java/org/jmhsrobotics/frc2025/commands/DriveTimeCommand.java

```diff
@@ -0,0 +1,46 @@
+package org.jmhsrobotics.frc2025.commands;
+
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import org.jmhsrobotics.frc2025.subsystems.drive.Drive;
+
+public class DriveTimeCommand extends Command {
+  private Drive drive;
+
+  private double driveSeconds;
+  private ChassisSpeeds speeds;
+
+  private Timer timer = new Timer();
+
+  public DriveTimeCommand(double seconds, double speed, Drive subsystem) {
+
```

Comment:
Please include units in the name for speed.
I.e rename to velocityMPs (Meters per Second)